util/pom.xml

```diff
@@ -83,5 +83,9 @@
       <groupId>com.google.truth</groupId>
       <artifactId>truth</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-api</artifactId>
+    </dependency>
```

Comment:
Why do we need this dep?

Reply:
We are using `io.grpc.Status` while extracting error type.